AppCenterData/AppCenterData/Internal/Util/MSDocumentUtils.m

```diff
@@ -129,7 +129,7 @@ + (MSDocumentWrapper *)documentWrapperFromDictionary:(NSObject *)object
                                                          innerError:nil
                                                             message:errorMessage];
     MSLogError([MSData logTag], @"Error deserializing data: %@.", [dataError localizedDescription]);
-    return [[MSDocumentWrapper alloc] initWithError:dataError documentId:nil];
+    return [[MSDocumentWrapper alloc] initWithError:dataError documentId:nil partition:nil eTag:nil];
```

Comment:
Same here the caller of `documentWrapperFromDictionary:documentType:fromDeviceCache` knows about the partition, if it's important we can pass it through.

Reply:
@clpolet Making this change is resulting in a cascade of other required changes. Okay to break these requests in to a different item?

Reply:
Can be done in a separate PR :).